### DISCUSSION TITLE
show medal rarity in medal popup

### DISCUSSION THREAD

**Discussion by EXtremeExploit** [Osekai/eclipse#160] [Ideas]
would be hella cool and very informative to know how rare a medal is in the popup
something like:
![image](https://user-images.githubusercontent.com/30161277/206939440-42cd16b0-5203-413c-b477-678f806ac48a.png)
or below the mods section. idk. seems a cool idea

**Comment by minusQuantumNeko** [comment]
I think I suggested it some time ago, but Tanza said no
